cwms-data-api/src/main/java/cwms/cda/data/dto/watersupply/WaterUserContract.java

```diff
@@ -144,21 +144,17 @@ public CwmsId getContractId() {
     @Override
     protected void validateInternal(CwmsDTOValidator validator) {
         super.validateInternal(validator);
-        validator.required(waterContract, "water-contract");
-        validator.required(contractEffectiveDate, "contract-effective-date");
-        validator.required(contractExpirationDate, "contract-expiration-date");
-        validator.required(contractedStorage, "contracted-storage");
-        validator.required(initialUseAllocation, "initial-use-allocation");
-        validator.required(futureUseAllocation, "future-use-allocation");
-        validator.required(storageUnitsId, "storage-units-id");
-        validator.required(futureUsePercentActivated, "future-use-percent-activated");
-        validator.required(totalAllocPercentActivated, "total-alloc-percent-activated");
-        validator.required(pumpOutLocation, "pump-out-location");
-        validator.required(pumpOutBelowLocation, "pump-out-below-location");
-        validator.required(pumpInLocation, "pump-in-location");
-        validator.required(contractId, "contract-id");
-        validator.required(waterUser, "water-user");
-
+        validator.required(getWaterContract(), "water-contract");
```

Comment:
use @JsonProperty(required=true) on the class variable instead of implementing validateInternal